app/setup.php

```diff
@@ -23,15 +23,19 @@
  * Theme setup
  */
 add_action('after_setup_theme', function () {
-    /**
-     * Enable features from Soil when plugin is activated
+
+	/**
+     * Enable features from the Soil plugin if activated.
+     * for Soil 4.x
      * @link https://roots.io/plugins/soil/
      */
-    add_theme_support('soil-clean-up');
-    add_theme_support('soil-jquery-cdn');
-    add_theme_support('soil-nav-walker');
-    add_theme_support('soil-nice-search');
-    add_theme_support('soil-relative-urls');
+    add_theme_support('soil', [
+        'clean-up',
+        'nav-walker',
+        'nice-search',
+        'relative-urls'
+    ]);
```

Comment:
It appears that the `soil-jquery-cdn` feature has been removed, was that intentional?

Reply:
Hm, I enable the same features as Sage 10 does.
Apparently the `soil-jquery-cdn` feature was removed from Soil, see this PR: https://github.com/roots/soil/pull/228